app/src/main/java/com/example/lapnguyen/weather_3/data/source/MainRepository.java

```diff
@@ -1,5 +1,7 @@
 package com.example.lapnguyen.weather_3.data.source;
 
+import com.example.lapnguyen.weather_3.data.model.Address;
```

Comment:
ERROR: Unused import: com.example.lapnguyen.weather_3.data.model.Address. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports